test/ship_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship.rb'
+require 'pry';
+
+
+class ShipTest < Minitest::Test
+
+  def test_ship_exsist
+
+    cruiser = Ship.new('cruiser', 3)
+
+    assert_instance_of Ship, cruiser
+  end
+
+  def test_it_has_readable_attributes
+
+    cruiser = Ship.new('cruiser', 3)
+
+    assert_equal 'cruiser', cruiser.ship_name
+    assert_equal 3, cruiser.length
+  end
+
+  def test_ship_health
+
+    cruiser = Ship.new('cruiser', 3)
+
+    assert_equal 3, cruiser.health
+  end
+
+  def test_if_ship_was_hit
+
+    cruiser = Ship.new('cruiser', 3)
+
+    assert_equal 2, cruiser.hit
```

Comment:
if I understand correctly, this should be evaluating `cruiser.health` since `cruiser.hit` doesn't actually have any return value? But you _would_ have to call `cruiser.hit` first.

Reply:
I see what youre saying. The reason I tested it this way is because I wanted to make sure that when I called ".hit" on a ship it would decrement the health var by 1. Even though it doesn't have a return value I wanted to make sure the method worked properly. Definitely down to discus this more because you do bring up an interesting point of it not having a return value.

Reply:
Looking at it I have no idea how `cruiser.hit` can be equal to `2`. Am I missing something?

Reply:
oh, I see! hit and health have the same return value since hit just reroutes to health!

Reply:
Yeah! When I push Ill add comments so that its clear for us in the future.